### PR TITLE
feat: add infra-did

### PR DESCRIPTION
**Pull request by pensivej**
Added did-resolver for infra did method.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -212,5 +212,11 @@
           "url": "http://driver-did-lit:8080/",
           "testIdentifiers": [ "did:lit:AEZ87t1bi5bRxmVh3ksMUi" ]
         }
```
Comment (BernhardFuchs) [inline config.json:214]: Comma missing.
Pls fix malformed json.

File: config.json [hunk 2]
```diff
@@ -212,5 +212,11 @@
...
+		{
+			"pattern": "^(did:infra:.+)$",
+			"url": "http://driver-did-infra:8080/",
+			"propertiesEndpoint": "true",
+			"testIdentifiers": [ "did:infra:01:PUB_K1_8PwG7of5B8p9Mpaw6XzeyYtSWJyeSXVtxZhPHQC5eZxZCkqiLU", "did:infra:coov:PUB_K1_8PwG7of5B8p9Mpaw6XzeyYtSWJyeSXVtxZhPHQC5eZxZCkqiLU"]
+		},
```
Comment (BernhardFuchs) [inline config.json:220]: Comma must be removed

**Comment by BernhardFuchs** [comment]
The container doesn't start and throws following error:
```
standard_init_linux.go:211: exec user process caused "exec format error"
```

Pls also fix the malformed `config.json` (see comments)

File: config.json (hunk 1 above)
Comment (pensivej) [reply config.json]: Yes, I fixed the comma.

File: config.json (hunk 2 above)
Comment (pensivej) [reply config.json]: Yes, I fixed the comma.

**Comment by BernhardFuchs** [comment]
Ok, thx for fixing `config.json`.  
The container still throws above error on startup.

**Comment by BernhardFuchs** [comment]
@pensivej Any updates on this?

**Comment by peacekeeper** [comment]
@pensivej Are you going to fix your driver and the PR? Otherwise we will probably close this soon.

**Comment by BernhardFuchs** [comment]
@pensivej We will close this PR due to inactivity.  
Feel free to reopen or create a new one, once the issues are fixed.

**Pull request closed without merging**